github/resource_github_repository.go

```diff
@@ -320,6 +320,7 @@ func resourceGithubRepository() *schema.Resource {
 			"vulnerability_alerts": {
 				Type:        schema.TypeBool,
 				Optional:    true,
+				Computed:    true,
```

Comment:
According to the [documentation](https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-behaviors#computed), "`Computed` is often used to represent values that are not user configurable or can not be known at time of `terraform plan` or `apply`, such as date of creation or a service specific UUID."

I suppose that `Computed` is appropriate here since it could be enabled by default at the org level.

Reply:
It is also required in combination with `optional` if an optional value will be persisted even if it's not been explicitly set.